monthly-exams/exam-1/src/App.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import './App.css';
+
+function App() {
+  let rainbow = ['red', 'green', 'blue', 'purple', 'orange', 'black'];
```

Comment:
You are not changing the value of `rainbow` so you can use `const` instead.
Btw, I do not think this is a rainbow spectrum so the variable should be named as simple as `const colors = [...]`